docker/pipelines_runner/pipelines_runner.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Copyright 2019 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+readonly google_cloud_project="${PROJECT}"
```

Comment:
The GOOGLE_CLOUD_PROJECT variable should be used for this: it's already automatically set in some environments like Cloud Shell

However, we shouldn't force the user to set it anyway if we can avoid it.  Given that you are mapping in $HOME/.config, the current project should be recoverable from gcloud

Reply:
Thank you Aaron! Updated. PTAL :)